### ISSUE TITLE
NVDA is reading the "Your Subscription" table as list.

### ISSUE THREAD

**Issue by msftedad** [msftedad/Azure-Toolkit-for-Eclipse#3]
NVDA is announcing the table as "list". When we traverse through the table, each cell is announced with its associated header column name, Row number etc. just like in tables. But defined using list.

**Test Environment:**
Windows 11 Enterprise Version:
NVDA (Version 2023.1), JAWS

**Pre-requisites**
Download JDK (version 17+) from [Java Archive Downloads - Java SE 17 (oracle.com)](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)

**Repro steps:**

1. Download "eclipse-accessibility-validation-1.0-SNAPSHOT-jar-with-dependencies.jar" provided in the attachments.
2. Make sure to install JDK (17+) before running above .jar file.
3. Turn on NVDA.
4. Check the behavior of NVDA while navigating through the table.

**Actual Behavior:**
NVDA is announcing the table as "list". When we traverse through the table each cell is announced with its associated header column name, Row number etc. just like in tables. But defined using list.

**Expected Behavior:**
It should be announced as only as Table by Screen Reader.

Attachments

[eclipse-accessibility-validation-1.0-SNAPSHOT-jar-with-dependencies-_1_.zip](https://github.com/msftedad/Azure-Toolkit-for-Eclipse/files/12257442/eclipse-accessibility-validation-1.0-SNAPSHOT-jar-with-dependencies-_1_.zip)

https://github.com/msftedad/Azure-Toolkit-for-Eclipse/assets/93735775/69ea4a23-562b-41d6-b129-5b7a91ccf982

**Comment by msftedad** [comment]
Hi team, Any update on this issue?
